<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>введите имя файла</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="fileTree">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>0</y>
      <width>421</width>
      <height>711</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnSearchFast">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search (Hash)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearchSlow">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search (Tree)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="searchResultList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreateFolder">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>btnCreateFolder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMountFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>btnMountFile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>btnDelete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
